test/integration/update.js

```diff
@@ -81,6 +81,45 @@ describe('Update file', () => {
     })
   })
 
+  describe.only('executable change', () => {
```

Comment:
oops

Reply:
Fixed, added a commit to avoid repeting this mistake in the future : https://github.com/cozy-labs/cozy-desktop/pull/1159/commits/5a1caa66f8b86baa07808d612d4103f3d642812d